scripts/Sample_Report.rmd.in

```diff
@@ -540,6 +541,8 @@ dd = lstats$Peak_Statistics$peaks_sample %>%
   mutate(mapped_total = as.numeric(mapped_total)) %>%
   mutate(value      = round(value/mapped_total,2))
 
+dd$sample_cnt <- sub("*.sorted.bam", "", dd$sample_cnt)
```

Comment:
this change can be directly added to the other modifications  using the pipe operator: 
```
... %>%
mutate(sample_cnt = gsub("*.sorted.bam", "", sample_cnt))
```

Reply:
Modified as suggested! Thanks.